data/config-schema.json

```diff
@@ -4,7 +4,7 @@
     "INodeConfig": {
       "description": "Detailed configuration of a node",
       "properties": {
-        "host": {
+        "ip": {
```

Comment:
nbd but we can keep this `host` (host vs. port)

Reply:
since we will auto generate stuff, let's just make our lives a bit simpler and always use separate host/port? we can make host optional in local configs, but this way we don't need to parse the host:port string ourselves.